client/components/aboutUs/styles.js

```diff
@@ -0,0 +1,123 @@
+import styled, { css } from "styled-components";
+import { breakpoint } from "../../frontend-config";
+
+import { Heading2, TextBody } from "../shared/styles";
+
+export const Wrapper = styled.div`
+  background: ${props => props.theme.colors.light};
+  padding: 30px 30px 50px;
+  width: 100%;
+  @media (min-width: ${breakpoint("lg")}) {
+    padding: 55px 30px;
+  }
+`;
+
+export const SubInfo = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  overflow: hidden;
+  padding: 55px 20px;
+
+  a {
+    color: ${props => props.theme.colors.link};
+  }
+
+  svg {
+    margin-bottom: 45px;
+    /* width: 40%; */
+  }
+
+  @media (min-width: ${breakpoint("xl")}) {
+    flex-direction: row;
+    gap: 60px;
+    svg {
+      margin-bottom: 0px;
+    }
+  }
+
+  ${props =>
+    props.grey &&
+    css`
+      background: ${props.theme.colors.grey};
+      flex-direction: column-reverse;
+      svg {
+        order: 2;
+        margin-bottom: 45px;
+      }
+      @media (min-width: ${breakpoint("xl")}) {
+        svg {
+          margin-bottom: 0px;
+        }
+      }
+    `}
+`;
+
+export const Logo = styled.img`
+  width: 231px;
+  height: 233px;
+  align-self: center;
+  margin-bottom: 64px;
```

Comment:
margin-bottom should probably removed on desktop, as it seems to align center with the text on the right